### PR TITLE
Add support for update-status as a reactive hook

### PR DESCRIPTION
**Pull request by johnsca**

### CODE REVIEW

**Comment by chuckbutler** [comment]
:+1:  LGTM

**Comment by marcoceppi** [comment]
:boat:

**Comment by mbruzek** [comment]
Actually Cory I think you need 3 backticks for the yaml code section to work.

File: README.md [hunk 1]
```diff
@@ -9,36 +9,43 @@ dependencies for use by the charm.
 # Usage
 
 To create a charm layer using this base layer, you need only include it in
-a ``layer.yaml`` file:
+a `layer.yaml` file:
 
-```yaml
+``yaml
```
Comment (mbruzek) [inline README.md]: You are missing backticks here, need three for a code block.

Comment (johnsca) [inline README.md]: Shoot, I got a little trigger-happy with the global-search-and-replace.  Thanks.

**Comment by mbruzek** [comment]
+1 LGTM

**Pull request merged**